pages/documentation/simulation/timedomain/dynawo.md

```diff
@@ -5,3 +5,98 @@ layout: default
 # Dynawo
 
 Powsybl provides an implementation to run dynamic simulations with [Dynawo](https://dynawo.github.io/).
+
+## Configuration
```

Comment:
Add a section above to explain how to get and install dynawo. Maybe we could git a link to the website or the readme where it's already explained

Reply:
One would need to install a distribution, https://github.com/dynawo/dynawo#distribution or https://dynawo.github.io/install/

Reply:
We have added a new section with a reference and a link to the website with the Dynawo distributions [https://dynawo.github.io/install/](https://dynawo.github.io/install/)